Flaky test on Windows

--- FAIL: TestQueryTimeout (0.05s)
    engine_test.go:107: expected timeout error but got none: expected error, got none

That test sometimes fail on windows (master branch).

To be investigated, but I think it is related to Window's being not precise enough with time.Time.

--- a/promql/engine_test.go
+++ b/promql/engine_test.go
@@ -99,7 +99,7 @@ func TestQueryTimeout(t *testing.T) {
 	defer cancelCtx()
 
 	query := engine.newTestQuery(func(ctx context.Context) error {
-		time.Sleep(50 * time.Millisecond)
+		time.Sleep(100 * time.Millisecond)
 		return contextDone(ctx, "test statement execution")
 	})
 

